pandas/core/indexing.py

```diff
@@ -206,7 +216,8 @@ def _setitem_with_indexer(self, indexer, value):
         self._has_valid_setitem_indexer(indexer)
 
         # also has the side effect of consolidating in-place
-        from pandas import Panel, DataFrame, Series
+        # TODO: Panel, DataFrame are not imported, remove?
```

Comment:
why is this added? these are not imported because it would be a circluar import otherwise (the ``NDFrame`` objects need to import the indexers to make them properties).